refinery/tool_manager/tests.py

```diff
@@ -2665,8 +2660,7 @@ def test_invalid_TLC_non_LIST_PAIR_file_relationships(self):
         with self.assertRaises(RuntimeError) as context:
             create_tool(tool_launch_configuration, self.user)
         self.assertIn(
-            "The `file_relationships` defined didn't yield a valid "
```

Comment:
Could we formulate this string to be more in line with 
```
"The 'file_relationships' {} type {} is not a valid LIST/PAIR nesting: {}".format(nesting_contents, nesting_types, valid_types)
```

Reply:
I'd prefer not to get into testing the precise wording of exceptions: too fragile.